www-servers/spawn-fcgi/spawn-fcgi-1.6.6.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit meson
+
+DESCRIPTION="A FCGI spawner for lighttpd and cherokee and other webservers"
+HOMEPAGE="https://redmine.lighttpd.net/projects/spawn-fcgi/wiki"
+SRC_URI="https://download.lighttpd.net/spawn-fcgi/releases-1.6.x/${P}.tar.xz"
+
+LICENSE="BSD GPL-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ppc ~ppc64 ~sparc ~x86"
+
+src_configure() {
+	meson_src_configure
```

Comment:
We can drop this as it's exported by `meson.eclass` and we're not defining anything special. I'll do that on merge. Thanks!